invenio_files_rest/models.py

```diff
@@ -1124,7 +1177,7 @@ def get(cls, bucket, key, version_id=None):
         return cls.query.filter(*filters).one_or_none()
 
     @classmethod
-    def get_versions(cls, bucket, key):
+    def get_versions(cls, bucket, key, desc=True):
```

Comment:
Actually, looking at the code, I would not do that. Since this is returning a query, the code that is using this method can filter/sort/whatever. I am removing it and fixing its usage.